go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -1177,3 +1190,21 @@ func (bnc *BaseNetworkController) wasPodReleasedBeforeStartup(uid, nad string) b
 	}
 	return bnc.releasedPodsBeforeStartup[nad].Has(uid)
 }
+
+func checkBridgeMapping(physicalNetworkName string) error {
+	external_ids, stderr, err := util.RunOVSVsctl("--if-exists", "get", "Open_vSwitch", ".",
+		"external_ids:ovn-bridge-mappings")
+	if err != nil {
+		return fmt.Errorf("failed to get ovn-bridge-mappings stderr:%s (%v)", stderr, err)
+	}
+
+	bridgeMappings := strings.Split(external_ids, ",")
+	for _, bridgeMapping := range bridgeMappings {
+		m := strings.Split(bridgeMapping, ":")
+		if m[0] == physicalNetworkName {
+			return nil
+		}
+	}
+
+	return fmt.Errorf("failed to find bridge mapping for physicalNetworkName: %s, external_ids %s", physicalNetworkName, external_ids)
```

Comment:
```suggestion
	return fmt.Errorf("failed to find bridge mapping for physicalNetworkName: %q, external_ids %s", physicalNetworkName, external_ids)
```

Reply:
And I wonder if we should print the name of the node ... I guess not.

Reply:
dont think so, after all it already on the pod that runs on that node